src/main/java/com/minecolonies/rankup/Rankup.java

```diff
@@ -66,19 +70,19 @@
     private final ConfigurationLoader<CommentedConfigurationNode> loader;
     private final SubInjectorModule subInjectorModule = new SubInjectorModule();
 
-    public EconomyService econ;
-    public static Rankup instance = null;
-    public        Game                                         game;
+    private PermissionsUtils perms;
+    private ConfigUtils      configUtils;
+    private AccountingUtils  accUtils;
+    public EconomyService   econ;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make econ a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)